WebDriverAgentLib/Categories/XCUIElement+FBForceTouch.m

```diff
@@ -29,9 +30,6 @@ - (BOOL)fb_forceTouchWithPressure:(double)pressure duration:(double)duration err
           @"ms": @(duration * 1000)
           }
       },
-    @{
```

Comment:
is release really not needed?

Reply:
it's not necessary in this implementation